<a href="https://colab.research.google.com/github/jinmeta/PYDS/blob/master/Done_JINENDRA_MEHTA_Practical_26(180570107070).ipynb" target="_parent"><img src="https://colab.research.google.com/assets/colab-badge.svg" alt="Open In Colab"/></a>

# Practical 26

**Name:** Jinendra Mehta

**Enrollment No:** 180570107070

**Sem & Class:** 5EC2

1) Download and Install Dataset of "country_wise_latest_covid19.csv"
    
     You can find this Dataset in your Google Classroom - Section: Popular DataSets
   
2) You can make it short name of the file for the easy remembering and programming purpose.
   
   Like: "country_wise_latest_covid19.csv" --> "covid.csv

===== Please Follow the instructions and perform all the tasks =====

Task 1: Print first five ROWS of the dataset.



In [ ]:
import pandas as pd
import numpy as np
import matplotlib as plt

df=pd.read_csv("/content/covid19.csv")
df.head()

Country  Confirmed  ...  1 week % increase                   WHOR
0  Afghanistan      36263  ...               2.07  Eastern Mediterranean
1      Albania       4880  ...              17.00                 Europe
2      Algeria      27973  ...              18.07                 Africa
3      Andorra        907  ...               2.60                 Europe
4       Angola        950  ...              26.84                 Africa

[5 rows x 15 columns]

Task 2: Print first eleven ROWS of the dataset.



In [ ]:
df.head(11)

Country  Confirmed  ...  1 week % increase                   WHOR
0           Afghanistan      36263  ...               2.07  Eastern Mediterranean
1               Albania       4880  ...              17.00                 Europe
2               Algeria      27973  ...              18.07                 Africa
3               Andorra        907  ...               2.60                 Europe
4                Angola        950  ...              26.84                 Africa
5   Antigua and Barbuda         86  ...              13.16               Americas
6             Argentina     167416  ...              28.02               Americas
7               Armenia      37390  ...               6.89                 Europe
8             Australia      15303  ...              23.13        Western Pacific
9               Austria      20558  ...               4.13                 Europe
10           Azerbaijan      30446  ...               9.16                 Europe

[11 rows x 15 columns]

Task 3: Print last five ROWS of the dataset.


In [ ]:
df.tail(5)

Country  Confirmed  ...  1 week % increase                   WHOR
44      Cuba       2532  ...               3.52               Americas
45    Cyprus       1060  ...               2.12                 Europe
46   Czechia      15516  ...              10.06                 Europe
47   Denmark      13761  ...               2.29                 Europe
48  Djibouti       5059  ...               0.78  Eastern Mediterranean

[5 rows x 15 columns]


Task 4: Print last eleven Print ROWS of the dataset.


In [ ]:
df.tail(11)

Country  Confirmed  ...  1 week % increase                   WHOR
38              Comoros        354  ...               5.99                 Africa
39  Congo (Brazzaville)       3200  ...              12.24                 Africa
40     Congo (Kinshasa)       8844  ...               4.75                 Africa
41           Costa Rica      15841  ...              37.34               Americas
42        Cote d'Ivoire      15655  ...               9.38                 Africa
43              Croatia       4881  ...              11.69                 Europe
44                 Cuba       2532  ...               3.52               Americas
45               Cyprus       1060  ...               2.12                 Europe
46              Czechia      15516  ...              10.06                 Europe
47              Denmark      13761  ...               2.29                 Europe
48             Djibouti       5059  ...               0.78  Eastern Mediterranean

[11 rows x 15 columns]


Task 5: Print only Column Name of the dataset as Index.


In [ ]:
df.columns

Index(['Country', 'Confirmed', 'Deaths', 'Recovered', 'Active', 'New cases',
       'New deaths', 'New recovered', 'Deaths / 100 Cases',
       'Recovered / 100 Cases', 'Deaths / 100 Recovered',
       'Confirmed last week', '1 week change', '1 week % increase', 'WHOR'],
      dtype='object')


Task 6: Print only "Country" column.


In [ ]:
df["Country"]

0                  Afghanistan
1                      Albania
2                      Algeria
3                      Andorra
4                       Angola
5          Antigua and Barbuda
6                    Argentina
7                      Armenia
8                    Australia
9                      Austria
10                  Azerbaijan
11                     Bahamas
12                     Bahrain
13                  Bangladesh
14                    Barbados
15                     Belarus
16                     Belgium
17                      Belize
18                       Benin
19                      Bhutan
20                     Bolivia
21      Bosnia and Herzegovina
22                    Botswana
23                      Brazil
24                      Brunei
25                    Bulgaria
26                Burkina Faso
27                       Burma
28                     Burundi
29                  Cabo Verde
30                    Cambodia
31                    Cameroon
32      


Task 7: Print ROW no. 5 to 11 from the dataset 
including all the column attributes (Hint use: iloc).


In [ ]:
df[5:12]

Country  Confirmed  ...  1 week % increase             WHOR
5   Antigua and Barbuda         86  ...              13.16         Americas
6             Argentina     167416  ...              28.02         Americas
7               Armenia      37390  ...               6.89           Europe
8             Australia      15303  ...              23.13  Western Pacific
9               Austria      20558  ...               4.13           Europe
10           Azerbaijan      30446  ...               9.16           Europe
11              Bahamas        382  ...             119.54         Americas

[7 rows x 15 columns]

In [ ]:
df.iloc[5:12]

Country  Confirmed  ...  1 week % increase             WHOR
5   Antigua and Barbuda         86  ...              13.16         Americas
6             Argentina     167416  ...              28.02         Americas
7               Armenia      37390  ...               6.89           Europe
8             Australia      15303  ...              23.13  Western Pacific
9               Austria      20558  ...               4.13           Europe
10           Azerbaijan      30446  ...               9.16           Europe
11              Bahamas        382  ...             119.54         Americas

[7 rows x 15 columns]


Task 8: Print ROW no. 5 in vertical with related 
attributes. (Hint use: iloc)

For Example: 
		
		Country: Angola
		Confirmed: 950	
		Deaths: 41
		Recovered: 242
		Active: 667
		New cases:18
		New deaths:	1
		New recovered: 0	
		Deaths / 100 Cases:	4.32
		Recovered / 100 Cases: 25.47
		Deaths / 100 Recovered:	16.94
		Confirmed last week: 749
		1 week change: 201
		1 week % increase:	26.84
		WHOR: Africa


In [ ]:
df.iloc[4]

Country                   Angola
Confirmed                    950
Deaths                        41
Recovered                    242
Active                       667
New cases                     18
New deaths                     1
New recovered                  0
Deaths / 100 Cases          4.32
Recovered / 100 Cases      25.47
Deaths / 100 Recovered     16.94
Confirmed last week          749
1 week change                201
1 week % increase          26.84
WHOR                      Africa
Name: 4, dtype: object


Task 9: Print the Dimension of Your Dataset/Dataframe. 

--> Answer: (49,15) 


In [ ]:
df.shape

(49, 15)


Task 10: Print only "Country" , "Deaths" and "Recovered" columns at one time.

 (i.e. Print only 3 columns) 

In [ ]:
df[['Country','Deaths','Recovered']]

Country  Deaths  Recovered
0                Afghanistan    1269      25198
1                    Albania     144       2745
2                    Algeria    1163      18837
3                    Andorra      52        803
4                     Angola      41        242
5        Antigua and Barbuda       3         65
6                  Argentina    3059      72575
7                    Armenia     711      26665
8                  Australia     167       9311
9                    Austria     713      18246
10                Azerbaijan     423      23242
11                   Bahamas      11         91
12                   Bahrain     141      36110
13                Bangladesh    2965     125683
14                  Barbados       7         94
15                   Belarus     538      60492
16                   Belgium    9822      17452
17                    Belize       2         26
18                     Benin      35       1036
19                    Bhutan       0         86
20                   Bolivia    2647      21478
21    Bosnia and Herzegovina     294       4930
22                  Botswana       2         63
23                    Brazil   87618    1846641
24                    Brunei       3        138
25                  Bulgaria     347       5585
26              Burkina Faso      53        926
27                     Burma       6        292
28                   Burundi       1        301
29                Cabo Verde      22       1550
30                  Cambodia       0        147
31                  Cameroon     391      14539
32                    Canada    8944          0
33  Central African Republic      59       1546
34                      Chad      75        810
35                     Chile    9187     319954
36                     China    4656      78869
37                  Colombia    8777     131161
38                   Comoros       7        328
39       Congo (Brazzaville)      54        829
40          Congo (Kinshasa)     208       5700
41                Costa Rica     115       3824
42             Cote d'Ivoire      96      10361
43                   Croatia     139       3936
44                      Cuba      87       2351
45                    Cyprus      19        852
46                   Czechia     373      11428
47                   Denmark     613      12605
48                  Djibouti      58       4977